config/22.1/make.conf

```diff
@@ -96,7 +96,7 @@ security_stunnel_SET=		TLS_LOG_IDENT
 security_suricata_SET=		HYPERSCAN NSS
 sysutils_msktutil_SET=		${PRODUCT_GSSAPI}
 www_lighttpd_UNSET=		LUA
-www_nginx_SET=			BROTLI NAXSI MAIL_IMAP MAIL_POP3 NJS VTS
+www_nginx_SET=			BROTLI NAXSI MAIL_IMAP MAIL_POP3 NJS VTS HEADERS_MORE
```

Comment:
Can you sort this in alphabetically please?

Reply:
done, sorry